Cargo.toml

```diff
@@ -112,8 +112,13 @@ features = ["parallel"]
 [target.'cfg(unix)'.dependencies]
 signal-hook = { version = "0.1.9", optional = true }
 
+[target.'cfg(not(target_arch="wasm32"))'.dev-dependencies]
+assert_cmd = { version = "1.0" }
```

Comment:
```toml
[target.'cfg(not(target_arch="wasm32"))'.dev-dependencies]
assert_cmd = { version = "1.0" }
```

I think I don't need this, when having it in a separate crate, do I?

Reply:
You should not.